lemur/certificates/views.py

```diff
@@ -1516,7 +1519,7 @@ def put(self, certificate_id, data=None):
     Certificates, "/certificates/<int:certificate_id>", endpoint="certificate"
 )
 api.add_resource(
-    Certificates, "/certificates/<int:certificate_id>/update/notify", endpoint="certificateUpdateNotify"
+    Certificates, "/certificates/<int:certificate_id>/update/switches", endpoint="certificateUpdateSwitches"
```

Comment:
I don't think it's likely that anyone's using this endpoint programmatically (i.e. not via the UI) right now -- and therefore I think this change is acceptable -- but I do at least want to call out that this is a breaking/backwards incompatible change if they are.

Reply:
Thanks @jtschladen, that's a great point; we could bump the version in the changelog for it if desired! Although as you noted, I also suspect this isn't a highly used API endpoint given that `PUT /certificates/<id>` has been around for much longer, and offers more flexibility to programmatic callers looking to update certs.

I'd be happy to preserve `/update/notify` and add a new, separate `/update/rotation` endpoint, if we want to be cautious. I _think_ I'd just need to add a new `Resource` class for it, split up the service funcs, and use a customPOST in the Angular service call.

Reply:
Thanks @havron for contributing the change. Code changes look good to me. We do suspect that it's not a highly used endpoint, so consolidating new changes in existing API makes sense. Given backward incompatibility, bumping version and adding changelog calling out this change will be helpful.

Reply:
Sounds good to me! I just added a changelog in https://github.com/Netflix/lemur/pull/3646/commits/3e94c84137a6a323deb955f54c6031e852a299a1, following the [new release process](https://lemur.readthedocs.io/en/latest/doing-a-release.html). Let me know if I can improve the release notes 🙂 @charhate

Reply:
Thanks @havron, really appreciate your efforts in compiling detailed release notes.
Given all the change going in next release and specially API change that is not backward compatible, the minor version change seems more appropriate (so 0.10.0 instead of 0.9.1). I have taken liberty to commit this https://github.com/Netflix/lemur/pull/3646/commits/7edee9f444e21b6463f55a4aa2e84c005fde80d4, hope you do not mind.
I'll merge the PR soon and do a release.